config/configStruct/otel.go

```diff
@@ -3,10 +3,15 @@ package configStruct
 import "fmt"
 
 type Otel struct {
-	Host string `env:"OTEL_HOST" envDefault:"localhost"`
-	Port int    `env:"OTEL_PORT" envDefault:"4317"`
+	Host   string `env:"OTEL_HOST" envDefault:"localhost"`
+	Port   int    `env:"OTEL_PORT" envDefault:"4317"`
+	Enable bool
 }
 
 func (o Otel) GetAddr() string {
 	return fmt.Sprintf("%s:%d", o.Host, o.Port)
 }
+
+func (o Otel) IsEnable() bool {
```

Comment:
It's unnecessary if it's only used to return `o.Enable`, we can use it directly if we need.

Reply:
However I can get this attribute 'Enable' throwing interface otelConfig. So how to do it is better?